recipe/meta.yaml

```
@@ -19,13 +20,13 @@ requirements:
   host:
     - python
     - pip
-    - numpy {{ numpy }}
+    - numpy >=2.0.0
```

```suggestion
    - numpy >=2.0.0,<2.3.0
```
see https://github.com/pydata/bottleneck/blob/v1.4.2/pyproject.toml#L16C5-L16C18